trousers/github_comment.template

```diff
@@ -6,17 +6,17 @@ PRbuilds results:
 {% endif %}
 
 {% if "exceptions" in results %}
-**Exceptions** ({{ results["exceptions"]["raw_output"].count("EXCEPTION") }})
+**{% if results["exceptions"]["raw_output"].count("EXCEPTION") == 0 %}&#x1F49A;{% else %}&#x1F525; (failed) {% endif %} **Exceptions**
 {{ " • ".join(links_for("thrown-exception")) }}
```

Comment:
I think the placement of the ```**```s are a bit off, you have one too many.

The line starts with ```**``` but also ends with ```**Exceptions**```. I think it should just be ```Exceptions**``` at the end.

Same for the other lines